### PR TITLE
feat: replace manage scripts with just

### PR DESCRIPTION
**Pull request by madwort**
* replace *-backend/manage.sh with justfile
* add bootstrap script to install just

current interface:

```
backend-server % just
Available recipes:
    build                     # build resources required to run tests
    build-test-image          # build lxd image for tests
    clean                     # remove temporary files relating to tests
    default
    install                   # install/update groups & system level configuration
    install-collector
    install-jobrunner
    install-osrelease
    install-packages          # install required system packages
    install-release-hatch
    lint                      # lint some shellscripts
    manage                    # install everything for a backend
    run_test target           # run a specific test
    test                      # run all tests
    testuser-key              # create ssh key for tests
    update-jobrunner commit_id="HEAD" # Update jobrunner to the specified commit_id & restart
    update-users
    whereami                  # report which backend configuration this justfile is using
```

### CODE REVIEW

**Comment by madwort** [comment]
nb. there's probably a good argument for renaming `just install` (formerly `scripts/install.sh`) to e.g. `install-system` and renaming `just manage` to `just install` - for discussion

**Review by bloodearnest — APPROVED** [review]
Left a suggestion about the bootstrap script.

One thing I'm not keen on in hindsight about this approach is using the same justfile for both dev and prod tasks. Feels very footgunny. e.g the dev commands don't require a BACKEND_JUST to work, but the prod ones do.

After this lands, maybe its worth trying to split the production commands into a separate justfile?

Not 100% sure if that would work, but I have an idea

File: justfile [hunk 1]
```diff
@@ -12,6 +13,115 @@ default:
...
+  @echo "Your current backend is: $BACKEND_JUST"
+
+update-users: check_backend_set
+  @{{ just_executable() }} update-users-$BACKEND_JUST
+
+[private]
+update-users-emis-backend:
+  ./scripts/update-users.sh developers emis-backend/researchers emis-backend/reviewers
+
+[private]
```
[the remark this answers is not in the document]
Comment (madwort) [reply justfile:51]: agree. ideally we'd move this config out of the justfile & into the backends' own directories - maybe e.g. a file called `emis-backend/users` that `update-users.sh` will read, with contents:
```
developers
emis-backend/researchers
emis-backend/reviewers
```
will create a follow-up ticket.

File: scripts/bootstrap.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+set -euo pipefail
+
+if test -z "$BACKEND_JUST"
+then
+  echo "Please specify your backend when running this file. For example:"
+  echo "BACKEND_JUST=test-backend ./bootstrap.sh";
+  exit 1
+fi
+
+echo "BACKEND_JUST=$BACKEND_JUST" > .env
```
[the remark this answers is not in the document]
Comment (madwort) [reply scripts/bootstrap.sh:11]: yep, happy to set via cli arg instead, will do that shortly in this PR. 

Making it system-wide is going to be useful, thinking to amend the contents from `emis-backend` to `emis`, which case I might make it system-wide in a follow-up *after* I've done that? to minimise faff?

**Pull request merged**